CuratedContent/EconomyOfOpenSourceSW.md

```diff
@@ -0,0 +1,28 @@
+# A look at the economic forces in Open Source Software
+
+The Slate Article *The Internet Was Built on the Free Labor of Open Source Developers. Is That Sustainable?* looks at the economics driving the Open Source movement and how that is putting those utilizing that software without investing in it at risk.
+
+Resource Information | Details
+:--- | :--- 
+Resource Name | "The Internet Was Built on the Free Labor of Open Source Developers. Is That Sustainable?"
+Website | https://www.vice.com/en_us/article/43zak3/the-internet-was-built-on-the-free-labor-of-open-source-developers-is-that-sustainable
+Focus | Funding open source software development.
+
+Much of modern HPC software is underpinned by Open Source - be that the [Linux Kernel](https://github.com/torvalds/linux), [Tensorflow](https://github.com/tensorflow/tensorflow), [SuperLU](https://github.com/xiaoyeli/superlu), or others. Yet treating the availability of that software as an externality means taking on the risk that said software has flaws that may impair your own work.
```

Comment:
The last sentance here culls out solely the risk of *flaws*. Is that the only risk concern treated in the article. I had the impression there were others...like the software going away for lack of support, etc.

Reply:
True - I have rewritten this about 3 times now. IU will try again.

Reply:
Ok, I don't mean to to re-write the whole article. Was just wondering about this specific sentence. May I suggest...

> Yet treating the availability of that software as an externality means taking on the associated risks such as show-stopper defects that may impair your own work or other more influential stakeholders pulling it in directions contrary to your own needs.

Reply:
Works for me